.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -5,6 +5,22 @@ dynamodb_CreateTable:
   synopsis: create a &DDB; table.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/dynamodb
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.dotnetv3.CreateTablesLoadDataExample
+        - sdk_version: 3
+          github: dotnetv3/dynamodb/FromSQL

```

Comment:
GitHub links must lead to a folder that contains a README, so all of the GH links in this file should go to `dotnetv3/dynamodb`.